contrib/win32/win32compat/ssh-agent/keyagent-request.c

```diff
@@ -179,6 +199,12 @@ process_add_identity(struct sshbuf* request, struct sshbuf* response, struct age
 	if ((success == 0) && reg && thumbprint)
 		RegDeleteKeyExA(reg, thumbprint, KEY_WOW64_64KEY, 0);
 
+	/* delete previous version of thumbprint keys without type */
```

Comment:
Isn't the logic should be before we create the thumbprint?

line 180 on the right hand side adds the thumbprint to registry.
Line 203 deletes the thumbprint added in line 180?